### DISCUSSION TITLE
My virtual host is not working correctly

### DISCUSSION THREAD

**Discussion by yeyomam** [leokhoa/laragon#799] [General]
Hello to the community. I recently created a local website with a custom domain using Laragon. I named the page mysite.local but then another day I wanted to name it test.local. Even though I delete the projects and recreate them, they both redirect me to another project mysite2.local. Any idea how to solve this mishap with laragon?

**Comment by Pen-y-Fan** [comment]
Check the Laragon properties:

![image](https://github.com/leokhoa/laragon/assets/40126936/70807bce-260f-4005-9932-3ad1d6448eac)

You may have changed your document root, the default is **c:\laragon\www** directory.

Laragon will automatically update the host file and create an Apache config based on the name of your folder in the **www** directory.

c:\laragon\www\mysite
c:\laragon\www\mysite2

Will create mysite.test & mysite2.test (unless you change the Hostname template to {name}.local

Check your Windows host file (click the h in the top right corner of Laragon) you should see:

```text
127.0.0.1      mysite.test       #laragon magic!   
127.0.0.1      mysite2.test       #laragon magic!  
```

You may need to approve changes to your host file, check for a shield icon in the task bar, and allow changes.

**Comment by DarAzizi** [comment]
disable your vpn
